tests/integration/request_test.py

```diff
@@ -100,3 +101,35 @@ def test_fuzzed_request(tag, id, mock_client):
     response = request.send()
 
     assert response.value == 'ok'
+
+
+@pytest.mark.parametrize(
+    'decorator_args, fuzzing_request_args',
+    [
+        (
+            {'tags': 'types'},
+            {'operation_id': 'get_expect_primitives', 'tag': 'types'},
+        ),
+        (
+            {},
+            {'operation_id': 'get_expect_primitives', 'tag': 'types'},
+        ),
+    ],
+)
+def test_post_fuzz_hook(mock_client, decorator_args, fuzzing_request_args):
+    def post_fuzz_hook(fuzzed_input):
+        new_input = fuzzed_input.copy()
```

Comment:
Huh. What's the purpose of the `.copy()` here?

Reply:
I was indecisive as to whether we should expect hooks to mutate the input dictionary or return a completely new dictionary. Reflecting on it, I think we should expect hooks to do the former, so I'll remove the `copy`.